go/vt/vtgate/planbuilder/plan_test.go

```diff
@@ -722,13 +726,37 @@ type testCase struct {
 	comments         string
 }
 
+var (
```

Comment:
we usually do this using the `TestOne` method and `onecate.txt` file. do we need both these approaches to test a single query?

Reply:
Oh I didn't know that. This just a made fixing bugs easier for me. Would you prefer I revert this?

Reply:
Reverted :+1: